Eatery Blue/Eatery Blue/UI/EateryScreen/EateryModelController.swift

```diff
@@ -154,7 +160,7 @@ class EateryModelController: EateryViewController {
                 presentMenuPicker()
             }
             addSpacer(height: 16)
-            addReportIssueView()
+            addReportIssueView(eateryID: Int(eatery?.id ?? 0))
```

Comment:
Is it a good idea to only add the ReportIssueView if `eatery.id` is non-nil?

Reply:
I think so because its possible to report an issue without it necessarily being related to a specific eatery